### PR TITLE
Gl Animation Loss Blank Animation Fix

### PR DESCRIPTION
**Pull request by epeach**
Removes the special case for drawing your own animation.

Prev - don't save anything to V3, but save a reference in the manifest. When loaded, look for this special case and build a new Piskel.

New - Blank animation in the animation library. Draw-your-own calls this animation and then treats edits to it like a normal animation library change.

Blank Animation in animation library is 'deleted' and retrieved via version id so it won't appear in student-facing animation library. The animation was found by manually saving and downloading blank 'drawn' animation. Can be found: cdo-animation-library/category_backgrounds/blank.png

cc: @poorvasingal

### CODE REVIEW

File: apps/src/gamelab/animationListModule.js [hunk 1]
```diff
@@ -321,31 +321,19 @@ export function setInitialAnimationList(serializedAnimationList) {
...
-        dataURI: null,
-        hasNewVersionThisSession: false
-      }));
-    dispatch(selectAnimation(key));
-    projectChanged();
-  };
+  return addLibraryAnimation(
+    {
+      name: 'animation',
+      sourceUrl: '/api/v1/animation-library/mUlvnlbeZ5GHYr_Lb4NIuMwPs7kGxHWz/category_backgrounds/blank.png',
```
Comment (islemaster) [inline apps/src/gamelab/animationListModule.js]: :joy: There's something wonderfully elegant about this.  It also might not be 100% intuitive.  Can we add a comment in `addBlankAnimation()` explaining this strategy of adding a secret, empty library animation?

File: apps/src/gamelab/animationListModule.js [hunk 2]
```diff
@@ -321,31 +321,19 @@ export function setInitialAnimationList(serializedAnimationList) {
...
+      frameSize: {x: 100, y: 100},
+      frameCount: 1,
+      looping: true,
+      frameDelay: 4,
+      version: 'mUlvnlbeZ5GHYr_Lb4NIuMwPs7kGxHWz',
+      loadedFromSource: true,
+      saved: false
```
Comment (islemaster) [inline apps/src/gamelab/animationListModule.js]: I suspect not all of these props are needed.  Let's double check the format for library animation props and conform to that.

File: apps/src/gamelab/animationListModule.js (hunk 1 above)
Comment (islemaster) [reply apps/src/gamelab/animationListModule.js]: Example library animation props.
```json
{
  "name": "bear",
  "frameCount": 1,
  "frameSize": {
    "x": 254,
    "y": 333
  },
  "looping": true,
  "frameDelay": 2,
  "jsonLastModified": "2017-04-18 05:56:09 UTC",
  "pngLastModified": "2017-04-18 05:56:10 UTC",
  "version": "vCJXAsYgUPWoNTt5h.OKY_tF3cEEY_Sf",
  "sourceUrl": "/api/v1/animation-library/vCJXAsYgUPWoNTt5h.OKY_tF3cEEY_Sf/category_animals/bear.png",
  "sourceSize": {
    "x": 254,
    "y": 333
  }
}
```

File: dashboard/test/ui/features/step_definitions/gamelab.rb [hunk 3]
```diff
@@ -44,6 +48,21 @@
...
+Then /^I select a blank animation$/ do
+  @browser.execute_script("$(\".uitest-animation-picker-item\")[0].click();")
+end
+
+Then /^I add a new, blank animation$/ do
+  steps <<-STEPS
+    And I open the animation picker
+    And I select a blank animation
+  STEPS
+end
```
Comment (islemaster) [inline dashboard/test/ui/features/step_definitions/gamelab.rb:64]: Are you also adding a UI test that uses these new steps?

Comment (epeach) [reply dashboard/test/ui/features/step_definitions/gamelab.rb:64]: Glad you pointed this out! I already wrote one but forgot to add it to the PR. Added in latest commit.

Comment (islemaster) [reply dashboard/test/ui/features/step_definitions/gamelab.rb:64]: :rofl:

**Pull request merged**